pkg/statdb/statdb.go

```diff
@@ -52,4 +52,12 @@ type NodeStats struct {
 	UptimeRatio        float64
 	UptimeSuccessCount int64
 	UptimeCount        int64
+	Wallet             string
+	Email              string
```

Comment:
imo, it would be better to add a `Meta` struct here rather than putting Wallet/Email directly in the `NodeStats` struct. Then you'd access Wallet with `nodeStats.MetaInfo.Wallet` or something. But what you have works if you don't wanna do that

Reply:
I ended up using pb.NodeOperator because it already has all the info we needed on it, I just didn't realize we already had that as an option.